<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soduku</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontal_Layout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_Layout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Hello&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Welcome to Sudoku&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;The history of Sudoku dates back to an 18th Century Swiss mathematician’s&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;game called “Latin Squares” (according to this article from the Economist)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;and some of the first number puzzles to appear in newspapers were published&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;in France in 1895. But the modern game of Sudoku as we recognize it today&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;was invented by Howard Garns, a freelance puzzle inventor from Connersville,&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;Indiana, USA in 1979 when it was published in &lt;/span&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; font-style:italic; color:#000000;&quot;&gt;Dell Pencil Puzzles and Word &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; font-style:italic; color:#000000;&quot;&gt;Games&lt;/span&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt; magazine. The puzzle was known as “Number Place,” since it involved&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Source Sans Pro','sans-serif'; font-size:16px; font-weight:700; color:#000000;&quot;&gt;placing individual numbers into empty spots on a 9 x 9 grid.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;This game is written by javad nematollahi and py-sudoku python library is&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;used for drawing the soduku numbers.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Help:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;help of sudoku will tell you how to play. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:700;&quot;&gt;Playing rules:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1.You just use these numbers in sudoku&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;[1,2,3,4,5,6,7,8,9]&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;2.In each row you should put one of 1 to 9 number and you cant set a number in two cell in a row. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;if you do this, the game will change the background color of that cell to red.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3.In each column you should put one of 1 to 9 number and you cant set a number in two cell in a&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;column. if you do this, the game will change the background color of that cell to red.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;4.In each square you should put one of 1 to 9 number and you cant set a number in two cell in a &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;square. if you do this, the game will change the background color of that cell to red.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;5. If you find the number of all empty cells, you will win the game.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 148, 185);</string>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>183</y>
      <width>245</width>
      <height>312</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(109, 109, 109);</string>
    </property>
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="Back_to_game"/>
    <addaction name="menunew"/>
    <addaction name="menuopenfile"/>
    <addaction name="solve_current_puzzle"/>
    <addaction name="About"/>
    <addaction name="Help"/>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menunew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="menuopenfile">
   <property name="text">
    <string>open file</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="solve_current_puzzle">
   <property name="text">
    <string>solve current puzzle</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Back_to_game">
   <property name="text">
    <string>Back to game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
